conbench/templates/index.html

```diff
@@ -30,12 +30,12 @@ <h3>Language-independent Continuous Benchmarking (CB) Framework</h3>
             <thead>
                 <tr>
                     <th scope="col">Run creation time</th>
-                    <th scope="col">Author</th>
+                    <th scope="col" class="author-col" style="width: 195px">Commit author</th>
```

Comment:
`author-col` is missing under `app.css` ?

Reply:
Thanks for noticing. I have removed `class="author-col"` here again, I think I wanted to do that before and forgot.